src/ctree-app/ctree-landing.html

```diff
@@ -331,10 +402,40 @@
           margin: auto;
           width: 75%;
         }
+
+        /* Mobile Feedback card styles */
+        .feedback-container {
+          background-color: #DBE8DE;
+          width: 65%;
+          box-shadow: none;
+        }
+
+        .feedback-cards {
+          padding-top: 5%;
+          padding-bottom: 5%;
+          display: flex;
+          flex-wrap: wrap;
+          flex-direction: column;
+        }
+        .feedback-card {
+          margin-top: 5%;
+          margin-bottom: 10%;
+          flex-basis: 0;
+          flex-grow: 1;
+          box-shadow: 0 2px 4px 0 rgba(0,0,0,0.5);
+          transition: 0.3s;
+          background-color: #FFFFFF;
+        }
+        .feedback-avatar {
+          margin-top: -10%;
+          border-radius: 50%;
+          box-shadow: 0 2px 4px 0 rgba(0,0,0,0.5);
+          max-width: 20%;
+        }
       }
 
       /* Section CSS Styling */
-      .section-styling {
+      .section-styling
```

Comment:
This looks like a bug to me.

Reply:
missing bracket, i'll add changes in next commit